Makefile

```diff
@@ -4,7 +4,7 @@ HOSTNAME=fortanix.com
 NAMESPACE=fortanix
```

Comment:
@aman-ahuja-fortanix @sanjeev-desai would we want to add something in our guide to reflect these changes

Also, have we tested this using the docker image as such 
docker run --rm -it --name terraform -v $(pwd):/workspace -v /etc/hosts:/etc/hosts -w /workspace hashicorp/terraform plan

Reply:
Why are we mounting /etc/hosts in the command

Reply:
This is the command how we can test dms binary along with official terrafrom docker image,  

sudo docker run --rm -it -v $PWD/terraform-provider-dsm:/root/.terraform.d/plugins/fortanix.com/fortanix/dsm/0.5.13/linux_amd64/terraform-provider-dsm  -v $PWD:/data -w /data hashicorp/terraform:latest plan


Output with above command:

Initializing the backend...

Initializing provider plugins...
- Finding fortanix.com/fortanix/dsm versions matching "0.5.14"...
- Installing fortanix.com/fortanix/dsm v0.5.14...
- Installed fortanix.com/fortanix/dsm v0.5.14 (unauthenticated)

Terraform has created a lock file .terraform.lock.hcl to record the provider
selections it made above. Include this file in your version control repository
so that Terraform can guarantee to make the same selections by default when
you run "terraform init" in the future.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
sanjeev@sanjeev:/terraform_data$ sudo docker run --rm -it -v $PWD/terraform-provider-dsm:/root/.terraform.d/plugins/fortanix.com/fortanix/dsm/0.5.14/linux_amd64/terraform-provider-dsm  -v $PWD:/data -w /data hashicorp/terraform:latest plan

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are needed.

Reply:
Above I see, 0.5.14 version but the committed one is 0.5.13.

Also, have we used terraform apply with the same command above to test a sample key/security object creation. Here, its just terraform plan.